lib/sequenceserver/routes.rb

```diff
@@ -108,6 +108,17 @@ class Routes < Sinatra::Base
       sequences.to_json
     end
 
+    post '/get_sequence' do
+      sequences = Sequence::Retriever.new(params["sequence_ids"].split(","),
+                                          params["database_ids"],
+                                          params["download"])
+
+      send_file(sequences.file.path,
+                :type     => sequences.mime,
+                :filename => sequences.filename) if params["download"]
+      sequences.to_json
```

Comment:
This line and the conditional in L118 is now redundant, right? You can simply pass `true` to `Sequence::Retriever` instead of `params['download']`.

 Similarly in `get '/get_sequence'` the `send_file` line, right?